torch/_decomp/__init__.py

```diff
@@ -437,8 +318,13 @@ def core_aten_decompositions() -> Dict[torch._ops.OperatorBase, Callable]:
 # to their default decomp. In old export, this will
 # be decomposed implicitly.
 def _decomp_table_to_post_autograd_aten():
+    from torch._export.utils import (
```

Comment:
Let me know if this sounds right to you in terms of the long term state:

(1) we don't need this extraa `torch._decomp._decomp_table_to_post_autograd_aten` API at all

(2) `torch._decomp.core_aten_decompositions()` above becomes a one-liner that just returns `torch.export.default_decompositions()`

If that sounds right, then is all of the complexity in this file about CIA ops a temporary state until we migrate?

Reply:
Yep this is just a table to make sure we don't break other ppl as there are some users who are still relying on post-autograd export behavior. And yes for (2), `torch._decomp.core_aten_decompositions()` will be an alias to `torch.export.default_decompositions()` cc @avikchaudhuri

Reply:
ok - in that case, looking forward to the cleanup PR :)

Reply:
Actually there are only like two places where this matters. I will just rewrite it there at their callsite.